src/components/BreadCrumbs/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import BackToLink from 'src/components/common/BackToLink';
+import Link from 'src/components/common/Link';
+
+export type Crumb = {
+  name: string; // name of crumb
+  to: string; // link to url/path
+};
+
+export type BreadCrumbsProps = {
+  hierarchical?: boolean; // if true, the 'to' paths are hierarchical and concantenated, default is false (each 'to' is a fully qualified path)
+  crumbs: Crumb[];
+};
+
+export default function BreadCrumbs({ hierarchical, crumbs }: BreadCrumbsProps): JSX.Element {
+  const theme = useTheme();
+  const breadCrumbs: Crumb[] =
+    hierarchical !== true
+      ? crumbs
+      : // construct the absolute paths using concatenation of previous paths
+        crumbs.reduce((acc, curr) => {
+          const crumb = {
+            name: curr.name,
+            to: [...acc.map((_, i) => crumbs[i].to), curr.to].join(''),
+          };
+          return [...acc, crumb];
+        }, [] as Crumb[]);
+
+  return (
+    <Box display='inline-flex' alignItems='center'>
+      {breadCrumbs.map((crumb: Crumb, index: number) => (
+        <>
+          {index === 0 ? (
+            <BackToLink id={`crumb_${index}`} to={crumb.to} name={crumb.name} />
```

Comment:
Safe to assume the first crumb always has "back-to-link" semantics?

Reply:
For now yes, our designs show this pattern. Should be easy to change if requirements change.